<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowMTPlanning</class>
 <widget class="QMainWindow" name="MainWindowMTPlanning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Threaded Planning</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <zorder>groupBox</zorder>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>181</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>Commands</string>
       </property>
       <widget class="QPushButton" name="pushButtonBuild">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>141</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>build Scene</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonReset">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>141</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>281</width>
         <height>271</height>
        </rect>
       </property>
       <property name="title">
        <string>Multi Threaded Planning</string>
       </property>
       <widget class="QComboBox" name="comboBoxColChecking">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>261</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxThreads">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>46</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Thread(s)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonPlanning">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Planning</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonPost">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Start Postpr.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonPlanningStop">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop Planning</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonPostStop">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>220</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop Postpr.</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>281</width>
         <height>241</height>
        </rect>
       </property>
       <property name="title">
        <string>Info</string>
       </property>
       <widget class="QLabel" name="labelThreads">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Threads_Working: 0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelThreadsIdle">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Threads_Idle: 0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelRuntime">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Runtime: 0.000s</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelOptiWork">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Opti_Threads_Working: 0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelOptiIdle">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Opti_Threads_Idle: 0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelOptiTime">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Optimizing Time: 0.000s</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
